crates/eurocode_1_1_2/src/section_3/equation_3_3.rs

```diff
@@ -0,0 +1,40 @@
+pub fn net_radiative_heat_flux_surface(
+    phi: f64,
+    epsilon_m: f64,
+    epsilon_f: f64,
+    sigma: f64,
+    delta_r: f64,
+    delta_m: f64,
```

Comment:
```suggestion
    theta_r: f64,
    theta_m: f64,
```
Same as before. Replace this throught the file